rkt/trust.go

```diff
@@ -59,7 +59,15 @@ func init() {
 func runTrust(cmd *cobra.Command, args []string) (exit int) {
 	if flagPrefix == "" && !flagRoot {
 		if len(args) != 0 {
-			stderr.Print("--root required for non-prefixed (root) keys")
+			trustName := args[0]
+			cmdName := "trust"
+			
+			stdout.Printf(`Prevented potentially dangerous unbounded trust of all containers signed by keys from %s
+To continue trusting this root domain:
+	rkt %s --root %s PUBKEY (local key file, no discovery)
+Or, establish a smaller scope with a prefix:
+	rkt %s --prefix "example.com/hello"
```

Comment:
@jonboulle I deleted the PUBKEY in the lines with the prefix. I hope you meant this in https://github.com/coreos/rkt/issues/480#issuecomment-220068592. But i am not sure if this is correct according to https://github.com/coreos/rkt/blob/master/Documentation/subcommands/trust.md#trust-a-key-from-specific-location